bazel/repositories.bzl

```diff
@@ -1317,7 +1317,8 @@ filegroup(
 def _com_github_fdio_vpp_vcl():
     external_http_archive(
         name = "com_github_fdio_vpp_vcl",
-        build_file_content = BUILD_ALL_CONTENT,
+        patch_cmds = ["mkdir _vcl && touch _vcl/out.txt"],
```

Comment:
this probably isnt necessary - i was trying to recreate the env/cmd from the original build